src/git-sync

```diff
@@ -46,6 +46,7 @@ function skippable {
 
 
 function steps {
+  echo_if_true "change_directory $(git_root)" "$IN_SUB_FOLDER"
```

Comment:
I think this is necessary for every command that syncs the main branch (extract, hack, ship, sync, and sync-fork). Would be nice if the cd was the first and last command (ie ran before preconditions)